src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -458,9 +458,7 @@ class BaseCouplingScheme : public CouplingScheme {
    *
    * The standard predictor is of order zero, i.e., simply the converged values
    * of the last time windows are taken as initial guess for the coupling iterations.
-   * Currently, an order 1 predictor (linear extrapolation) and order 2 predictor
-   * (see https://doi.org/10.1016/j.compstruc.2008.11.013, p.796, Algorithm line 1 )
-   * is implement besides that.
+   * Currently, an order 1 predictor (linear extrapolation) is implement besides that.
```

Comment:
Please rewrite this comment. Feels like a simple thing is explained in a complicated way.

Reply:
Done [cbd8c32](https://github.com/precice/precice/pull/1503/commits/cbd8c329b3a688218805858e71400e042d2319cb)